Sources/XcodeGenKit/PBXProjGenerator.swift

```diff
@@ -71,7 +71,7 @@ public class PBXProjGenerator {
         }
 
         let buildConfigs: [XCBuildConfiguration] = spec.configs.map { config in
-            let buildSettings = spec.getProjectBuildSettings(config: config)
+            let buildSettings = spec.getProjectBuildSettings(basePath: basePath, config: config)
```

Comment:
I'm thinking of moving the basePath into the spec itself, so we won't need things like this 👍

Reply:
I've done this in #103